<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_gestaofornecedores</class>
 <widget class="QDialog" name="frm_gestaofornecedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>682</width>
     <height>630</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestão Fornecedores</string>
    </attribute>
    <widget class="QTableWidget" name="tw_ge_listarFornecedores">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>661</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ge_editar">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>570</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>23</y>
       <width>88</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filtrar por</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_filtro"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>47</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txt_ge_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Novo Fornecedor</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>462</x>
       <y>565</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btn_nv_novo">
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_nv_gravar">
        <property name="text">
         <string>Gravar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>6</y>
       <width>663</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Empresa</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>31</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>CNPJ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_cnpj"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>61</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Numero</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_numeroRua"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>UF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_uf"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>166</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Cidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_cidade"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Porte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_novoPorte"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>241</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ocupação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_ocupacao"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>171</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Rua</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_rua"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Bairro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_bairro"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>90</y>
        <width>131</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>CEP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_cep"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>231</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Razão Social</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_razaoSocial"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>241</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Nome Fantasia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nomeFantasia"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Telefone1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_tel1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Telefone2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_tel2"/>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>layoutWidget_2</zorder>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
